rebuild_slashing_protection.py

```diff
@@ -269,13 +274,11 @@ def process_chain(head, justified_epoch, processed_blocks, best_attestations_in_
 justified_epoch = int(finality_checkpoints["current_justified"]["epoch"])
 processed_blocks = []
 best_attestations_in_chains = []
+proposal_slots = []
```

Comment:
We only need the highest observed proposal slot
```suggestion
proposal_slot = 0
```

Reply:
Removing this suggestion in favor of the list solution